projects/js-packages/charts/src/components/pie-chart/pie-chart.tsx

```diff
@@ -138,14 +138,15 @@ const PieChartInternal = ( {
 
 	const width = size;
 	const height = size;
+	const adjustedHeight =
+		showLegend && legendAlignmentVertical === 'top' ? height - legendHeight : height;
```

Comment:
This tests fine for me, but I don't understand why the vertical alignment makes a difference... it will take up vertical space no matter if it's at the top or the bottom.